src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robot1 = {};
```

Comment:
robot2,3,4,5.... robotN... 

how to understand what is stored in each of these variables?